extractor/src/test/java/org/schabi/newpipe/extractor/services/bandcamp/BandcampStreamExtractorTest.java

```diff
@@ -0,0 +1,98 @@
+// Created by Fynn Godau 2019, licensed GNU GPL version 3 or later
+
+package org.schabi.newpipe.extractor.services.bandcamp;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.schabi.newpipe.DownloaderTestImpl;
+import org.schabi.newpipe.extractor.NewPipe;
+import org.schabi.newpipe.extractor.exceptions.ExtractionException;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.services.bandcamp.extractors.BandcampStreamExtractor;
+import org.schabi.newpipe.extractor.stream.StreamExtractor;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.schabi.newpipe.extractor.ServiceList.bandcamp;
+
+public class BandcampStreamExtractorTest {
+
+    private static BandcampStreamExtractor extractor;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        NewPipe.init(DownloaderTestImpl.getInstance());
+        extractor = (BandcampStreamExtractor) bandcamp
+                .getStreamExtractor("https://zachbenson.bandcamp.com/track/kitchen");
+        extractor.fetchPage();
+    }
+
+    @Test(expected = ExtractionException.class)
+    public void testAlbum() throws ExtractionException {
+        bandcamp.getStreamExtractor("https://zachbenson.bandcamp.com/album/prom");
+    }
```

Comment:
Remove this test. It should be handled by returning `false` in `onAcceptUrl()` in the appropriate `LinkHandler`